rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -526,9 +525,10 @@ def _combine_sparse_dense_features(
             else:
                 dense_features.append(f)
 
-        output = tf.concat(dense_features, axis=-1) * mask
-        # apply mean to convert sequence to sentence features
-        output = tf.reduce_sum(output, axis=1) / tf.reduce_sum(mask, axis=1)
+        # extract the CLS token feature vector
```

Comment:
let's extract the `cls` token before applying `dense_layer_for_sparse`, then we save the computation over the entire sequence